core/src/main/resources/application.conf

```diff
@@ -84,6 +84,10 @@ system {
 
   // Since the WorkflowLogCopyRouter is initialized in code, this is the number of workers
   number-of-workflow-log-copy-workers = 10
+
+  // Whether to run the metadata summarizer in this Cromwell instance.  There should only be one metadata summarizer
+  // per database, so if this Cromwell instance is being used in read-only mode this value should be `false`.
```

Comment:
@mcovarr I had a few lines saying why I thought it was a great idea until I thought of the thing after "OTOH". If the right nomenclature could be found to capture what it's really doing and get over my pedantic streak I'd be for that myself